app/assets/javascripts/components/sidebar/sidebar_recent.es6.jsx

```diff
@@ -14,11 +14,18 @@ class SidebarRecent extends Component {
   // --------------------------------------------------
   // Styles
   // --------------------------------------------------
+
   get styles() {
     return {
-      label: {
-        paddingLeft: '12px',
-        fontSize: StyleConstants.fonts.sizes.smaller,
+      container: {
+        display: 'flex',
+        flexFlow: 'column',
+      },
+      border: {
+        marginTop: '14px',
+        marginLeft: '14px',
```

Comment:
The margin's should be `16px` now, and we don't need `marginLeft` anymore if you merge master.